test/test.dart

```diff
@@ -388,27 +388,22 @@ void xtest(String? name, dynamic Function() testFunction, {dynamic skip, Map<Str
 }
 
 BaasHelper? baasHelper;
-Map<String, String?> _testArgs = {};
-
-Future<void> setupTests(List<String>? args) async {
-  _testArgs = parseTestArguments(args);
-  testName = _testArgs["name"];
 
+void setupTests() {
   setUpAll(() async {
-    baasHelper = await BaasHelper.setupBaas(_testArgs);
-  });
+    baasHelper = await BaasHelper.setupBaas();
 
-  setUp(() {
     Realm.logger = Logger.detached('test run')
```

Comment:
This seems to have been moved from setUp to setUpAll - is this intentional? This way, if we replace the logger in a test, wouldn't that logger continue to be used in subsequent tests?

Reply:
Yes, the logger is static to the isolate anyway. It will not be torn down regardless. Notice how the log tests use `Isolate.run` to not mess with the primary logger. We might as well move it here.

Reply:
This doesn't sound correct to me - maybe I'm missing something, but the Realm Core logger is very much a singleton, so setting it up on a different isolate just ends up replacing it.

Reply:
I'm not sure if something has happened in my absence. But the Dart side logger is per isolate, and the static core side logger will forward messages to each isolate logger and keep track of the log level of each.

Reply:
Hm, I'll try and take a look.

Reply:
Yeah, you're right - I had completely forgotten about how this was done in Dart. Let's keep it like this then.